WikiTemplate/Crypto/wikitemplate-cryptoDesktop.md

```diff
@@ -0,0 +1,150 @@
+
+## Panel
+
+- [ ] Verify when wallet is not created opening the wallet panel shows `Learn More` message. Clicking on it opens `brave://wallet` in a new tab
+- [ ] Verify when wallet is created, opening the panel shows default account and selected network and account balance
+- [ ] Verify Buy/Send/Swap buttons are available and loads corresponding screens to perform action
+- [ ] Verify transaction history button is shown and lists recent transactions for selected network when clicked
+- [ ] Verify changing network in panel updates the corresponding account in panel and in portfolio page
+- [ ] Verify changing account in panel updates the corresponding network in panel and in portfolio page
+- [ ] Verify when a Dapp is connected to wallet account, panel shows a `Connected` state notification
+- [ ] Verify when a Dapp connection is revoked, panel shows a `Disconnected` state notification
+- [ ] Verify selecting `Connected Sites` from panel menu loads `brave://settings/content/<Network>` for Ethereum/Solana based on selected account
+- [ ] Verify selecting `View on Block explorer` from panel loads the corresponding account's details on the network block explorer
+- [ ] Verify selecting `Help Center` opens a new tab and loads `https://support.brave.com/hc/en-us/articles/4415497656461-Brave-Walllet-FAQ`
+- [ ] Verify you are able to lock the wallet from panel menu
+- [ ] Verify clicking on the expand button on the top-left of panel loads the wallet page in a new tab
+
+## Portfolio & Accounts
+
+- [ ] Verify after creating a wallet it loads portfolio screen by default
+- [ ] Verify portfolio page shows balance of all accounts
+- [ ] Verify able to show/hide balance on portfolio screen
+- [ ] Verify able to search for token in portfolio screen
+- [ ] Verify able to filter assets list by specific network via the network dropdown
+- [ ] Verify able to sort NFT's and Assets list by balance amount
+- [ ] Verify able to add or remove assets from view via `Visible assets` button
+- [ ] Verify able to add a new custom asset for specific network
+- [ ] Verify able to add hide/remove custom asset via `Visible assets` list
+- [ ] Verify able to create a new primary account for Ethereum/Solana/Filecoin via `Create Account` in accounts tab
+- [ ] Verify able to import Ethereum/Solana/Filecoin accounts via `Import` option
+- [ ] Verify imported accounts are listed under `Secondary Accounts`
+- [ ] Verify able to import Ethereum/Solana/Filecoin accounts via `Import from hardware wallet`
+- [ ] Verify imported hardware wallet accounts are listed under `Secondary Accounts`
+- [ ] Verify Wallet backup message is shown on both portfolio and accounts tab if skipped during onboarding
+- [ ] Verify Brave Wallet is not default message is shown when MetaMask is installed and `Brave Wallet (Prefer extensions)` is set as default wallet provider
+
+## Ethereum
+
+- [ ] Verify when a new wallet is created it creates an Ethereum account by default
```

Comment:
Looks like we now set Solana as the default account

Reply:
Yeah need to update it. Had this done before Solana fix